pkg/reconciler/revision/resources/deploy.go

```diff
@@ -169,8 +181,23 @@ func makePodSpec(rev *v1.Revision, cfg *config.Config) (*corev1.PodSpec, error)
 
 	// If concurrencyStateEndpoint is enabled, add the serviceAccountToken to QP via a projected volume
 	if cfg.Deployment.ConcurrencyStateEndpoint != "" {
+		// add token for audience "concurrency-state-hook" under filename ConcurrencyStateTokenFilename
+		addToken(tokenVolume, queue.ConcurrencyStateTokenFilename, concurrencyStateHook, ptr.Int64(600))
```

Comment:
Can we give a const value to define the expire time(600) and next(3600)?

Reply:
@jwcesign 
It seems as if the convention that Knative adopted is not to create constants in such a case.  A quick search of ptr.Int64 on the code base shows many such cases not made into constants

I believe that @dprotaso indicated in the past that he justifies constants only if the same value needs to appear in more than one place - which is not the case here. 

Let's keep the same style here as we use elsewhere.